src/components/vars.sass

```diff
@@ -159,7 +159,7 @@ $bottom-nav-expanded-height: 130px
 $left-nav-contracted-width: 94px
 $left-nav-expanded-width: 700px
 
-$right-nav-contracted-width: 94px
+$right-nav-contracted-width: 100px
```

Comment:
This adjustment inspired me to review Michael's spec again.  There are a few places where borders, shadows, etc. are off.  At some point we might want to bug Michael for the latest spec and sync things up.

Reply:
I did this here because for Dataflow they want "Shared Work" (instead of "Class Work") which didn't fit without the few extra pixels.

Reply:
Right, and I did test some longer strings here and even if we have to use multiple lines in the future, it still looks pretty good.  Also, this answers my question above about what tabs are needed for DF!